rtc_data_service/http_server/config/default.toml

```diff
@@ -10,6 +10,12 @@ debug = false
 lib_path = "/root/rtc-data/rtc_auth_enclave/build/bin/enclave.signed.so"
 attestation_provider_url = "https://sharedeus.eus.attest.azure.net"
 
+[exec_enclave]
+debug = false
+# Relative to the binary
+lib_path = "/root/rtc-data/rtc_exec_enclave/build/bin/enclave.signed.so"
+attestation_provider_url = "https://sharedeus.eus.attest.azure.net"
```

Comment:
Would there ever be a case where we'll use different attestation providers in the same configuration context?

If not, we should maybe queue a task to factor this out somewhere more global?

Reply:
Yep, the attestation provider depends on the region the code is running in, so it depends on the runtime context.